.test-infra/jenkins/job_LoadTests_Java.groovy

```diff
@@ -203,6 +203,201 @@ def loadTestConfigurations = [
                         numWorkers          : 5,
                         autoscalingAlgorithm: "NONE"
                 ]
+        ],
+        [
```

Comment:
If we add new elements to `loadTestConfiguration` we will still have 1 Jenkins job but 2x more steps in it. I belive we need separate `loadTestConfigurations` for both batch job and streaming job. Could you modify this accordingly? 

Also, as we discussed offline, the `loadTestConfiguration` could be a "template" for the options that could further be parametrized in jobs. This will prevent repeating too much code.

Reply:
You're right, the template approach is much more elegant and maintainable. I applied your suggestions